### PR TITLE
ticket 3 complete

### PR DESCRIPTION
**Pull request by yassmine-x**

### CODE REVIEW

File: __tests__/news.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+describe("GET/api/topics", () => {
+  test("status:200 responds with all the topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body; // deconstructing key of topics from body
+        expect(Array.isArray(topics)).toBe(true);
+        expect(topics).toHaveLength(3);
+        expect(
```
Comment (daviidbartlett) [inline __tests__/news.test.js:19]: this test isn't doing anything. you're not expecting it to be anything

File: controllers/news.controller.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const fetchTopics = require("../models/news.model");
+
+const getTopics = (req, res) => {
+  console.log("hellp!");
```
Comment (daviidbartlett) [inline controllers/news.controller.js]: remove console.log

File: controllers/news.controller.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  fetchTopics().then((response) => {
+    console.log(response, "hey!!");
```
Comment (daviidbartlett) [inline controllers/news.controller.js]: remove

File: models/news.model.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection.js");
+
+const fetchTopics = () => {
+  return db.query("SELECT * FROM topics").then(({ rows: topics }) => {
+    console.log(topics);
```
Comment (daviidbartlett) [inline models/news.model.js]: remove

File: controllers/news.controller.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    res.send(response);
```
Comment (daviidbartlett) [inline controllers/news.controller.js]: I'd put in an object like `{topics:[...]}`

**Comment by yassmine-x** [comment]
I forgot to push my changes to ticket 4 and now I have tickets 3 and 4

**Pull request closed without merging**